src/pages/Home.jsx

```diff
@@ -36,48 +71,77 @@ const Home = () => {
-              title="Secure Storage"
-              description="Store vital documents like IDs, Social Security information, birth certificates, medical records, and bank statements in a valid digital format."
-            />
-            <KeyFeatures
-              isReallySmallScreen={isReallySmallScreen}
-              icon={<Diversity1Icon fontSize={isReallySmallScreen ? 'medium' : 'large'} />}
-              title="Nonprofit-Caseworker Integration"
-              description="The platform facilitates smooth communication between nonprofit organizations, case workers, and the individuals they serve. It allows nonprofit organizations to maintain a contact list, and caseworkers are assigned contacts whose data they can access securely."
-            />
-            <KeyFeatures
-              isReallySmallScreen={isReallySmallScreen}
-              icon={<SupportIcon fontSize={isReallySmallScreen ? 'medium' : 'large'} />}
-              title="Support Service"
-              description="Verified documents can be used to facilitate access to service such as housing support and shelter accommodation. The platform simplifies the process of submitting necessary documents for such services."
-            />
+            <Box
+              height="calc(100vh - 64px)"
+              display="flex"
+              flexDirection="column"
+              justifyContent="center"
+              alignItems="center"
+            >
+              {logoSection}
+              <Typography
+                variant={isReallySmallScreen ? 'h4' : 'h3'}
+                component="p"
+                fontWeight="600"
+                mb={isReallySmallScreen ? 8 : 18}
+                color="primary"
+              >
+                Personal Access System for Services
+              </Typography>
+              <Typography color="secondary.main" sx={{ fontSize: '24px' }}>
+                Learn More
+              </Typography>
+              <IconButton aria-label="Scroll down" color="secondary" onClick={handleClick}>
+                <ArrowCircleDownOutlinedIcon sx={{ fontSize: '32px' }} />
+              </IconButton>
+            </Box>
+            <div ref={aboutRef}>
+              <HomeSection
+                isReallySmallScreen={isReallySmallScreen}
+                componentImageSrc="/assets/web-security-green.png"
+                componentImageAlt=""
+                title="Keep Your Documents Safe and Secure Using Decentralized Technology"
+                description="Our innovative solution empowers individuals to manage their critical documents and control access for trusted organizations. PASS simplifies service access, enabling seamless documents requests and secure data sharing for a smoother support process."
+                button="Request a Demo"
+                href="mailto:hugh@codeforpdx.org"
+                hasMargin
+                ref={aboutRef}
```

Comment:
Andy caught this. Looks like there's a duplicate `ref` on Line 111, causing error in the browser console (see clip).

https://github.com/codeforpdx/PASS/assets/14917816/37ec1076-72fa-4108-893b-cebed08734a3

Error should disappear once we get that removed (see end of clip above).

Reply:
so this is where I had ref setup previously! I don't know what all happened there, but removed now.